src/github.com/getlantern/flashlight/client/config.go

```diff
@@ -19,8 +18,6 @@ type ClientConfig struct {
 	FrontedServers []*FrontedServerInfo
 	ChainedServers map[string]*ChainedServerInfo
 	MasqueradeSets map[string][]*fronted.Masquerade
-
-	ProxiedSites *proxiedsites.Config // List of proxied site domains that get routed through Lantern rather than accessed directly
```

Comment:
I moved proxiedsites.Config from client into the base config.  This is to keep the client from reconfiguring itself needlessly, since it actually doesn't care about the proxied sites.